lib/user.js

```diff
@@ -31,9 +33,23 @@ const instanceMethods = {
         });
     },
 
-    callFunction(name, args) {
+    callFunction(name, args, service = undefined) {
+        function cleanArgs(args) {
+            for (const arg of args) {
+                if (typeof arg === "object") {
+                    for (const [key, value] of Object.entries(arg)) {
+                        if (value === undefined) {
+                            delete arg[key];
+                        }
+                    }
+                }
+            }
+            return args;
+        }
+        args = cleanArgs(args);
+
         return new Promise((resolve, reject) => {
-            this._callFunction(name, args, (result, error) => {
+            this._callFunction(name, args, service, (result, error) => {
```

Comment:
this does not resolve

Reply:
ah.. ignore. the github ui have failed me

Reply:
I have a patch locally that makes this just `return promisify(cb => this._callFunction(name, args, service, cb));` so you won't have to worry about that in the future :)